src/index.js

```diff
@@ -701,7 +726,7 @@ class CampaignStandardCoreAPI {
    */
   getAllCustomResources (customResource, parameters) {
     const sdkDetails = { customResource, parameters }
-
+    logger.warn('getAllCustomResources has been deprecated, use getAllProfileAndServicesExt()')
```

Comment:
I think we should mention the two new functions. "use either X or Y". Your instruction is correct if they want the same behaviour, but not correct if they want what they think getAllCustomResources should do (if they never used it before). Same in the @deprecated notice